buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -168,6 +168,33 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
+      "displayName": "Create Helix Test Jobs",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "c6c4c611-aa2e-4a33-b606-5eaba2196824",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "solution": "src\\tests.builds",
```

Comment:
This seems like a potential failure point I'd rather not deal with down the road as it may be difficult to repro on a dev machine vs just using something in your enlistment.  I thought we used to run the VS Developer Command Prompt script as part of build.cmd, but I guess that changed.